contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -239,6 +239,21 @@ def _verify_height(self, prev_click_height: int, post_click_height: int, track_n
                 track_name, prev_click_height, post_click_height))
 
 
+class CollapseOrExpandAllTracks(CaptureWindowE2ETestCaseBase):
```

Comment:
I find the name of this a bit misleading - does it expand or collapse? Maybe rename this to something like "ToggleCollapsedStateOfAllTracks"?

Reply:
jep, thanks, that's better